src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
```

Comment:
Don't use literals in expressions (HARDCODE). Instead use constants explaining this values